test_unstructured/partition/test_text_type.py

```diff
@@ -294,3 +294,19 @@ def test_item_titles():
 )
 def test_is_us_city_state_zip(text, expected):
     assert text_type.is_us_city_state_zip(text) is expected
+
+
+@pytest.mark.parametrize(
```

Comment:
This test looks good, the rest of the test file changes I think can be reverted.